package.json

```
@@ -89,6 +89,7 @@
               "codellama/CodeLlama-13b-hf",
               "Phind/Phind-CodeLlama-34B-v2",
               "WizardLM/WizardCoder-Python-34B-V1.0",
+              "deepseek-ai/deepseek-coder-6.7b-base",
```

New naming for templates:
```suggestion
              "hf/deepseek-ai/deepseek-coder-6.7b-base",
```

---

Thanks, rebased and updated all to get the `hf/` prefix in